walkthroughs/howto-k8s-tls-acm/README.md

```diff
@@ -0,0 +1,224 @@
+- color-route-blue: matches on HTTP header "blue" to route traffic to virtual node `blue`
+- color-route-green: matches on HTTP header "green" to route traffic to virtual node `green` 
+- color-route-red: matches on HTTP header "red" to route traffic to virtual node `red`
+
+Virtual nodes `blue` and `green` are configured with TLS enabled for their respective listeners. Here's the spec for `green` Virtual Node:
+
+```
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualNode
+metadata:
+  name: green
+  namespace: howto-k8s-tls-file-based
+spec:
+  podSelector:
+    matchLabels:
+      app: color
+      version: green
+  listeners:
+    - portMapping:
+        port: 8080
+        protocol: http
+      healthCheck:
+        protocol: http
+        path: '/ping'
+        healthyThreshold: 2
+        unhealthyThreshold: 2
+        timeoutMillis: 2000
+        intervalMillis: 5000
+      tls:
+        mode: STRICT
+        certificate:
+          acm:
+            certificateARN: arn:aws:acm:us-west-2:<ACCOUNT_ID>>:certificate/<certificate>
+  serviceDiscovery:
+    dns:
+      hostname: color-green.howto-k8s-tls-file-based.svc.cluster.local
+```
+
+The `tls` block specifies the ACM certificate to use. See [Transport Layer Security (TLS)](https://docs.aws.amazon.com/app-mesh/latest/userguide/tls.html) for the IAM permissions that are required while using ACM PCA for TLS.
+
+
+## Setup 4: Verify TLS is enabled
+
+```bash
+kubectl -n default run -it --rm curler --image=tutum/curl /bin/bash
+```
+
+```
+curl -H "color_header: blue" front.howto-k8s-tls-file-based.svc.cluster.local:8080/; echo;
+
+curl -H "color_header: green" front.howto-k8s-tls-file-based.svc.cluster.local:8080/; echo;
+```
+
+You should see a successful response when using the HTTP header "color_header: blue"
+
+Let's check the SSL handshake statistics.
+
+```bash
+BLUE_POD=$(kubectl get pod -l "version=blue" -n howto-k8s-tls-file-based --output=jsonpath={.items..metadata.name})
+kubectl exec -it $BLUE_POD -n howto-k8s-tls-file-based -c envoy -- curl -s http://localhost:9901/stats | grep ssl.handshake
+```
+
+You should see output similar to: listener.0.0.0.0_15000.ssl.handshake: 1, indicating a successful SSL handshake was achieved between front and blue color app
+
+
+## Step 5: Cleanup
+
+If you want to keep the application running, you can do so, but this is the end of this walkthrough.Run the following commands to clean up and tear down the resources that we’ve created.
+
+```bash
+kubectl delete -f _output/manifest.yaml
+```
+
+And finally delete the certificates.
+```bash
+aws acm delete-certificate --certificate-arn $BLUE_CERTIFICATE_ARN
+aws acm delete-certificate --certificate-arn $GREEN_CERTIFICATE_ARN
+aws acm-pca update-certificate-authority --certificate-authority-arn $ROOT_CA_ARN --status DISABLED
+aws acm-pca delete-certificate-authority --certificate-authority-arn $ROOT_CA_ARN
+```
+
```

Comment:
I realize this isn't unique to this walkthrough, but maybe call out a one-liner to delete the repo in ECR, as well?

Reply:
Added commands for deleting the ECR repos